modules/cnf-about-numa-aware-scheduling.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-numa-aware-scheduling_{context}"]
+= About NUMA-aware scheduling
+
+Non-Uniform Memory Access (NUMA) is a compute platform architecture that allows different CPUs to access different regions of memory at different speeds. NUMA resource topology refers to the locations of CPUs, memory, and PCI devices relative to each other in the compute node. Co-located resources are said to be in the same NUMA zone. For high-performance applications, pod workloads should be processed in a single NUMA zone.
+
+NUMA architecture allows a CPU with multiple memory controllers to use any available memory across CPU complexes, regardless of where the memory is located. This allows for increased flexibility at the expense of performance. A CPU processing a workload using memory that is outside its NUMA zone is slower than a workload processed in a single NUMA zone. Also, for I/O-constrained workloads, the network interface on a distant NUMA zone slows down how quickly information can reach the application. High-performance workloads, such as telecommunications workloads, cannot operate to specification under these conditions. To process latency-sensitive or high-performance workloads efficiently, all the requested cluster compute resources (CPUs, memory, devices) should be aligned in the same NUMA zone. NUMA-aware scheduling also improves pod density per compute node for greater resource efficiency.
+
+The default {product-title} pod scheduler makes scheduling decisions based on available resources of the entire compute node, not individual NUMA zones. If the most restrictive resource alignment is requested in the kubelet topology manager, error conditions can occur when admitting the pod to a node. Conversely, if the most restrictive resource alignment is not requested, the pod can be admitted to the node without proper resource alignment, leading to worse or unpredictable performance. For example, runaway pod creation with `Topology Affinity Error` statuses can occur when the pod scheduler makes suboptimal scheduling decisions for guaranteed pod workloads by not knowing if the pod's requested resources are available. Scheduling mismatch decisions can cause indefinite pod startup delays. Also, depending on the cluster state and resource allocation, poor pod scheduling decisions can cause extra load on the cluster because of failed startup attempts.
```

Comment:
> makes scheduling decisions

ISG calls this "anthropomorphizing," and it is not a fan of it. :) 

Not crucial, but consider rephrasing.